docs/semgrep-supply-chain/license-compliance.md

```diff
@@ -5,69 +5,62 @@ title: License compliance
 
-This section provides guides on blocking or allowing packages in CI pipelines based on the license of a package.
+By default, all licenses are set to **Allow**. You must configure your policies to block or leave comments on licenses.
 
-### Types of license policies
+## View license policy
 
-Licenses in Semgrep are assigned the following policies:
+To view a package's license policy:
 
-<dl>
-<dt>Allow</dt>
-<dd>Packages with licenses assigned this type of permission are allowed for use in the codebase.</dd>
-<dt>Comment</dt>
-<dd>Packages with licenses assigned this type of permission are allowed for use in the codebase. A comment is added to the PR or MR that introduces the package into the codebase. This permission can be useful when you want to remind or warn developers to use certain licenses for internal use only.</dd>
-<dt>Block</dt>
-<dd>Packages with licenses assigned this type of permission are <strong>not</strong> allowed into the codebase. A comment is added to the PR or MR that introduces the package into the codebase.</dd>
-</dl>
+1. [Sign in to Semgrep AppSec Platform](https://semgrep.dev/login).
+2. Navigate to **[Supply Chain](https://semgrep.dev/orgs/-/supply-chain)** > **Dependencies**.
 
-:::tip
-By default, all licenses are set to **Allow**. You must configure your policies to block or leave comments on licenses.
-:::
+## Change the license policy
 
 To change the policies of packages based on the license:
 
-1. From the **Supply Chain** page, click **License configuration** on the header menu.
+1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login), and navigate to **Supply Chain > License configuration.
 2. Browse the available licenses within the **License configuration** section.
-![License configuration section](/img/sc-license-configuration.png#md-width)
-_**Figure**. The Supply Chain > Settings > License configuration section._
-3. Click the permission (**Allow**, **Comment**, or **Block**) you want to set the license to.
-4. Optional: Block entire categories of licenses by clicking on the **Set all to** drop-down box next to the license category.
+   ![License configuration section](/img/sc-license-configuration.png#md-width)
+   _**Figure**. The Supply Chain > Settings > License configuration section._
+1. Click the permission (**Allow**, **Comment**, or **Block**) you want to set the license to.
+2. Optional: Block entire categories of licenses by clicking on the **Set all to** drop-down box next to the license category.
 
-### License categories
+## License categories
 
-This section describes license categories and licenses identified by Semgrep Supply Chain.
+Semgrep Supply Chain can identify the following licenses and license categories.
 
-#### Popular weak-copyleft licenses
+### Weak-copyleft licenses
```

Comment:
FYI - the use of "Popular" is to match the UI itself:

![image](https://github.com/user-attachments/assets/96d92b4a-5517-4f2d-9761-cc5471a87140)